samples/scaleout/Version_6/Shared/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NServiceBus" version="5.2.14" targetFramework="net45" />
```

Comment:
There is no point in running a distributor in a pure V6 environment. The only reason is that you want you cluster to talk to V5 endpoints. In such case you reuse the old V5 distributor.

The pure V6 scale out with client distribution will have to be a separate sample because it is completely different approach. This is the issue I use to track the pure V6 scale out sample: https://github.com/Particular/docs.particular.net/issues/799